conformance/google-java/src/test/kotlin/com/connectrpc/conformance/Conformance.kt

```diff
@@ -168,14 +187,17 @@ class Conformance(
                 for (res in stream.resultChannel()) {
                     res.maybeFold(
                         onCompletion = { result ->
-                            // For some reason we keep timing out on these calls and not actually getting a real response like with grpc?
-                            assertThat(result.code).isEqualTo(Code.RESOURCE_EXHAUSTED)
-                            assertThat(result.connectError()!!.code).isEqualTo(Code.RESOURCE_EXHAUSTED)
-                            assertThat(result.connectError()!!.message).isEqualTo("soirée 🎉")
-                            assertThat(result.connectError()!!.unpackedDetails(ErrorDetail::class)).containsExactly(
-                                expectedErrorDetail
-                            )
-                            countDownLatch.countDown()
+                            try {
```

Comment:
Wouldn't we want this to explode if any of the asserts fail rather than allowing tests to continue?

Reply:
It will still fail the test, but this will prevent it waiting 5 seconds on the countDownLatch for the failure to register. Verified by adding a failing assert.